proxy/healthy_endpoints_test.go

```diff
@@ -26,6 +26,39 @@ func defaultEndpointRegistry() *routing.EndpointRegistry {
 	})
 }
 
+func sendGetRequest(t *testing.T, ps *httptest.Server, consistentHashKey int) (*http.Response, error) {
+	req, err := http.NewRequest("GET", ps.URL, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Since you are using assert you may also use `require.NoError(t, err)` for consistency.